packages/manager/modules/pci/src/projects/onboarding/onboarding.html

```diff
@@ -0,0 +1,14 @@
+<div class="pci_projects_onboarding">
+    <div class="row">
+        <div class="col-md-6">
+            <h1 data-translate="pci_projects_onboarding_title"></h1>
+            <p data-translate="pci_projects_onboarding_description1"></p>
+            <p data-translate="pci_projects_onboarding_description2"></p>
+            <p data-translate="pci_projects_onboarding_description3"></p>
+            <oui-button class="mt-4" variant="primary">
+                <span data-translate="pci_projects_onboarding_create_project"></span>
+            </oui-button>
+        </div>
+    </div>
+    <img data-ng-src="{{ $ctrl.illustration }}" alt="" />
```

Comment:
No extra self-closing tag.

```suggestion
    <img data-ng-src="{{ $ctrl.illustration }}" alt="">
```